src/main/java/org/booking/controllers/FlightController.java

```diff
@@ -7,39 +7,47 @@
 import org.booking.enums.FilePath;
 import org.booking.interfaces.IController;
 import org.booking.services.ServiceFlight;
+import org.booking.utils.Console;
 import org.booking.utils.FileWorker;
 import org.booking.utils.Randomize;
 
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class FlightController implements IController {
     private final ServiceFlight service = new ServiceFlight();
 
-    private long generateTime() {
-        return System.currentTimeMillis();
+    private List<Long> generateTime(int count) {
```

Comment:
Now we have DateUtil to work with date. Can you change this method using our utilities?